src/main/java/uk/gov/hmcts/reform/blobrouter/tasks/processors/BlobProcessor.java

```diff
@@ -63,14 +63,17 @@ public void process(BlobClient blobClient) {
 
         logger.info("Processing {} from {} container", blobName, containerName);
 
-        envelopeService.saveEvent(containerName, blobName, Event.FILE_PROCESSING_STARTED);
+        BlobLeaseClient leaseClient = leaseClientProvider.get(blobClient);
+        try {
+            leaseClient.acquireLease(60);
+        } catch (Exception exc) {
```

Comment:
Maybe we should catch some specific exception, that represents a conflict or sth? We still might have problems connecting and that should result in logging an error.

Reply:
done